tests/bin/assigner/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(assigner_test assigner_test.cpp)
```

Comment:
This test should check `runner` but not `assigner`, so better name `runner_test`